smdebug/profiler/utils.py

```diff
@@ -300,3 +300,11 @@ def stop_smdataparallel_profiler(smdataparallel, base_dir):
         ensure_dir(new_file_name)
         if os.path.exists(smdataparallel_temp_file):
             shutil.move(smdataparallel_temp_file, new_file_name)
+
+
+def get_last_modified_time(filepath):
+    """
+    Get the last time that the file at the given filepath was modified, in the form of a datetime object.
+    """
+    last_modified_time = os.path.getmtime(filepath)
+    return datetime.fromtimestamp(last_modified_time)  # get the last time the config was modified
```

Comment:
I want to make sure this does not return false positives.

What if an agent owned by the platform team touches the file which causes changes in the `last_modified_time` value?

Checking for a change in file_size itself might provide a stronger signal.
What do you think?

Reply:
Interesting. I see you've used getatime below and getmtime here.

What is the difference between the two?

Reply:
Are there any long term risks? Do we have to worry about the OS the container is running on?